CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 # set C standard to gnu99
 set(CMAKE_C_STANDARD 99)
 
+# Set OpenCL version explicitly (quiets a build warning)
+add_definitions(-DCL_TARGET_OPENCL_VERSION=300)
```

Comment:
ROCm has only very recently started supporting OpenCL 3.0, I think we'll need to set this to 2.2 to allow the outrigger builds to work.

Reply:
We don't need anything in 3.0, so let's keep at 2.2 for simple compatibility.